### PR TITLE
ask whatever leisure=dog_park is wheelchair accessible [generalize, change icon]

### LINKED ISSUE #1252: ask whatever leisure=dog_park is wheelchair accessible
### General
Affected tag(s) to be modified/added: [wheelchair](https://wiki.openstreetmap.org/wiki/Key:wheelchair)
Question asked: **Is dog park wheelchair accessible?**

### Checklist
Checklist for quest suggestions (see guidelines):
- [x] 🚧 To be added tag is established and has a useful purpose
- [x] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
- [x] 🐿️ Easily answerable by everyone from the outside but a survey is necessary 
- [x] 💤 Not an overwhelming percentage of elements have the same answer (No spam)
- [x] 🕓 Applies to a reasonable number of elements (Worth the effort)

### Ideas for implementation

Should be fairly easy to base it on existing wheelchair quests (note that this one can be enabled by default).

**Element selection:** 

**Metadata needed:** No

**Proposed GUI:**

Should be fairly easy to base it on existing wheelchair quests.

**Other:**

https://github.com/simonpoole/beautified-JOSM-preset/issues/114 is a related issue (I was not 100% sure whatever tagging wheelchair on leisure=dog_park is a good idea).

### PR DESCRIPTION
**Pull request by matkoniecz**
fixes #1252 

Unfortunately, it ended far more dog park specific than I expected, as typical dog park has no name.

### CODE REVIEW

**Comment by matkoniecz** [comment]
My initial reason for making it a separate quest was that interior wheelchair quest is disabled by default, and this one can be done without entering buildings.

Though anyway it can be generalized a bit with description changed based on tags.

**Comment by westnordost** [comment]
Hmm right. Are there now other places that are outside for which it is reasonable to ask for wheelchair accessibility? Graveyards? Gardens?

**Comment by matkoniecz** [comment]
No idea, but I also was not expecting that dog parks are a case where this tag is considered as useful.

**Comment by westnordost** [comment]
Is it considered useful? I mean, only because there is
- hgv=yes/no
- car=yes/no
- bus=yes/no
- goods=yes/no
- horse=yes/no
- bicycle=yes/no
- foot=yes/no
- etc etc.
in JOSM presets for each and every street, doesn't make it considered useful to set all these values.

**Comment by matkoniecz** [comment]
I am doing this based mostly on recommendation from https://github.com/simonpoole/beautified-JOSM-preset/issues/114 (I was open and I am still open to making PR to either SC or Vespucci preset)

**Comment by westnordost** [comment]
Hm okay, I'll scan through this JOSM preset file and see where else wheelchair is given as "mandatory", compare it with the list in StreetComplete.

**Comment by westnordost** [comment]
Indeed, I did not find any more public-and-outside things that could be included here.

But anyway, please rename to AddWheelchairPublicOutside or something like that and use the same question as for the other quest ("Is this place accessible... ?") so that it can easily be extended for other places as well.

File: app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessDogPark.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	{
+		String wheelchair = answer.getString(AddWheelchairAccessOutsideForm.ANSWER);
+		if(wheelchair != null)
+		{
+			changes.add("wheelchair", wheelchair);
+		}
+	}
+
+	@Override public String getCommitMessage() { return "Add wheelchair access"; }
+	@Override public int getIcon() { return R.drawable.ic_quest_wheelchair_shop; }
```
Comment (westnordost) [inline app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessDogPark.java]: Ah, also, this should have the wheelchair with the green background. (You do not need re-export anything, just copy the shop wheelchair icon and change the color of that one path in the xml)

File: app/src/main/res/drawable/ic_quest_wheelchair_outside.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (rugk) [inline app/src/main/res/drawable/ic_quest_wheelchair_outside.xml]: Is there an SVG version of this quest icon?

**Comment by westnordost** [comment]
No, I just copied the business one and replaced the background color

Am 4. Dezember 2018 09:09:28 MEZ schrieb rugk <notifications@github.com>:
>rugk commented on this pull request.
>
>
>
>> @@ -0,0 +1,53 @@
>+<vector xmlns:android="http://schemas.android.com/apk/res/android"
>
>Is there an SVG version of this quest icon?

**Comment by rugk** [comment]
Ah, so as far as I see, it is the same version as the toilet wheelchair quest now uses?


image

**Comment by westnordost** [comment]
Ah, yes

Am 4. Dezember 2018 19:48:12 MEZ schrieb rugk <notifications@github.com>:
>Ah, so as far as I see, it is the same version as the toilet wheelchair
>quest now uses?
>
>
>image

**Pull request merged**